docs/src/math_framework.md

```diff
@@ -74,6 +74,58 @@ while the _contravariant basis_ is the opposite: gradient in ``x`` of the coordi
 \mathbf{e}^i = \nabla_x \xi^i
 ```
 
+Here is an intuitive example of how the _contravariant basis_ and _covariant basis_ changes when a coordinate is being transformed.
```

Comment:
Hi @yiyuef , thank you for your contributions! I would suggest that you rephrase some sentences a bit.


```suggestion
Here is a visual representation of how vectors can be represented in _contravariant_ and _covariant_ bases.
```